src/pyOpenMS/pxds/AASequence.pxd

```diff
@@ -36,89 +36,89 @@ cdef extern from "<OpenMS/CHEMISTRY/AASequence.h>" namespace "OpenMS":
         String toUnmodifiedString() nogil except +
 
         # returns the peptide as string without any modifications
-        String toUniModString() nogil except +
+        String toUniModString() nogil except + # wrap-doc:returns the peptide as string with UniMod-style modifications embedded in brackets
 
         String toBracketString() nogil except + #wrap-doc:create a TPP compatible string of the modified sequence using bracket notation. Uses integer mass by default.
```

Comment:
This is a bit nitpicking, but maybe we could standardize if we use `# warp-doc` or `#wrap_doc` (at least per-file basis)

Reply:
Hey @oliveralka,
Yes, I am using `# warp-doc` everywhere.

Reply:
Could you please change it, if you see that it is done differently in the same file e.g. in line 41.

Reply:
Sure.